docs/features/kubernetes/troubleshooting.md

```diff
@@ -0,0 +1,99 @@
+---
+id: troubleshooting
+title: Troubleshooting Kubernetes
+sidebar_label: Troubleshooting
+description: Troubleshooting for Kubernetes
+---
+
+## Kubernetes is not showing up on Service Entities
+
+Sometimes, kubernetes is not showing up on service entities, we should test your
+k8s cluster are already connected to backstage or not.
+
+```curl
+# curl request
```

Comment:
thinking you could remove this since the context of this sample code will be clear?